src/lib.rs

```diff
@@ -92,3 +91,13 @@ pub use crate::hasher_impl::sha3::{Keccak224, Keccak256, Keccak384, Keccak512};
 pub use crate::hasher_impl::sha3::{Sha3_224, Sha3_256, Sha3_384, Sha3_512};
 #[cfg(feature = "strobe")]
 pub use crate::hasher_impl::strobe::{Strobe256, Strobe512, StrobeHasher};
+
+/// Code reserved for Identity "hash"
+pub const IDENTITY_CODE: u64 = 0x0;
+/// Helper for generating Identity hashes (context https://github.com/multiformats/rust-multihash/pull/196).
+/// Will error if `data.len() > S`
+///
+/// See examples for a few other approaches if this doesn't fit your application
+pub fn identity_hash<const S: usize>(data: impl AsRef<[u8]>) -> Result<MultihashGeneric<S>> {
+    MultihashGeneric::wrap(IDENTITY_CODE, data.as_ref())
+}
```

Comment:
It seems inconsistent to me to provide only this here when we are showing two ways to do it in the example.

Why have this at all? Seems trivial to implement so I am not sure what we are gaining here. At the same time, we are adding an item to our public API which in the future might cause a breaking change.